### PR TITLE
 [MISC] Use more meaningful group naming for multi-group tests

### PR DESCRIPTION
**Pull request by lucasgameiroborges**
VM port of https://github.com/canonical/postgresql-k8s-operator/pull/707

### CODE REVIEW

**Review by a-velasco — APPROVED** [review]
lgtm! Left a comment on the [k8s PR](https://github.com/canonical/postgresql-k8s-operator/pull/707#pullrequestreview-2327966826).

File: tests/integration/new_relations/test_new_relations.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 INVALID_EXTRA_USER_ROLE_BLOCKING_MESSAGE = "invalid role(s) for extra user roles"
 
 
-@pytest.mark.group(1)
+@pytest.mark.group("main_tests")
```
Comment (dragomirp) [inline tests/integration/new_relations/test_new_relations.py:40]: Something like high availability might be a more meaningful label.

**Review by dragomirp — APPROVED** [review]

File: tests/integration/new_relations/test_new_relations.py (hunk 1 above)
Comment (taurus-forever) [reply tests/integration/new_relations/test_new_relations.py:40]: ha_tests?

**Review by taurus-forever — APPROVED** [review]
LGTM, thank you taking care! IMHO, you can update to MySQL as well (talk to Paulo). ;-)

**Review by marceloneppel — APPROVED** [review]

File: tests/integration/new_relations/test_new_relations.py (hunk 1 above)
Comment (lucasgameiroborges) [reply tests/integration/new_relations/test_new_relations.py]: oops, this is different from K8s: we're on `test_new_relations`, not `self_healing` - what about `new_relations_tests`?

Comment (lucasgameiroborges) [reply tests/integration/new_relations/test_new_relations.py]: merging this so it gets out of the way - we can always create follow ups if better ideas come!

**Pull request merged**